.github/workflows/main.yml

```diff
@@ -0,0 +1,256 @@
+name: 'main'
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+
+  build-binaries:
+    runs-on: ubuntu-latest
+    env:
+      CC: gcc-9
+      CXX: g++-9
+
+    steps:
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update -qq
+        sudo apt install -y g++-9 build-essential cmake tclsh ant default-jre swig google-perftools libgoogle-perftools-dev python3 python3-dev uuid uuid-dev tcl-dev flex libfl-dev
+
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 1
+
+    - uses: actions/setup-python@v2
+      with:
+         python-version: '3.7'
+
+    - name: Build binaries
+      run: |
+        cd Surelog && make PREFIX=$PWD/../image/ release install -j $(nproc) && cd ..
+        make PREFIX=$PWD/image install
+        make -C vcddiff PREFIX=./image
+        cp -p vcddiff/vcddiff image/bin/vcddiff
+        git clone https://github.com/antmicro/verilator.git -b add-ci
+        cd verilator/Surelog && git submodule update --init --recursive
```

Comment:
Why is Verilator cloned here, but `vcddiff` is a submodule? One is useless without the other, so I think having both as submodules or both cloned by scripts would make more sense. I imagine the way it is set up now could be confusing for newcomers.

Reply:
That's right. I added verilator as a submodule.